modelci/cli/modelhub.py

```diff
@@ -242,3 +244,194 @@ def delete(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.delete(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         if r.status_code == HTTPStatus.NO_CONTENT:
             typer.echo(f"Model {model_id} deleted")
+
+
+@app.command('convert')
+def convert(
+        src: Optional[str] = typer.Argument(..., help='the prototype model framework to be converted'),
+        dst: Optional[str] = typer.Argument(..., help='the destination model framework after converting'),
+        model_path: Optional[str] = typer.Argument(..., help='path of the model to be converted'
+                                                   'such as \'/home/model\' for keras savedmodel or'
+                                                   '\'/home/model/xxx.suffix\' for a file'),
+        model_name: Optional[str] = typer.Option(..., '-n', '--name', help='model name'),
+        version: Optional[int] = typer.Option(..., '-v', '--version', min=1, help='Version number'),
+
+        shape_input: Optional[str] = typer.Option(None, '-si', '--shapei', help='input shape such as [-1,28,28]'),
+        dtype_input: Optional[str] = typer.Option(None, '-di', '--dtypei', help='input data type such as \'float\' '
+                                                                                'or special types like \'tf.float32\''),
+        format_input: Optional[str] = typer.Option(None, '-fi', '--formati', help='input format for tensorrt such as'
+                                                                                  '\'FORMAT_NONE\' \'FORMAT_NHWC\' '
+                                                                                  '\'FORMAT_NCHW\' '),
+
+        shape_output: Optional[str] = typer.Option(None, '-so', '--shapeo', help='output shape such as [-1, 1000]'),
+        dtype_output: Optional[str] = typer.Option(None, '-do', '--dtypeo', help='output data type such as \'float\''),
+
+        model_task: Optional[str] = typer.Option(..., '-t', '--task', help='task of model'
+                                                                           'such as \'IMAGE_CLASSIFICATION\''
+                                                                           'or \'SEGMENTATION\'\'OBJECT_DETECTION\'')
+):
+    """
+    TODO replace input and output str with IOShape
+    """
+    import modelci.hub.converter.converter as cvt
```

Comment:
converter should be converter, I'm quite not understand what is `cvt` means, no one will blame you to add 7 extra  words.